src/Test.Utilities/DiagnosticAnalyzerTestBase.cs

```diff
@@ -162,6 +165,12 @@ protected static DiagnosticResult GetCSharpResultAt(int line, int column, Diagno
             return GetResultAt(CSharpDefaultFilePath, line, column, rule, messageArguments);
         }
 
+        protected static DiagnosticResult GetCSharpResultAt(IEnumerable<Tuple<int, int>> lineColumnPairs, DiagnosticDescriptor rule, params object[] messageArguments)
```

Comment:
>GetCSharpResultAt [](start = 42, length = 17)

Please also add the VB overload for consistency. Also file a test issue to add VB tests (at least few sanity ones to make sure it works).